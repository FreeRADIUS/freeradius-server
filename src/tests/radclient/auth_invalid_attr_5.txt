#
#	ARGV: -i 123 -c 1 -x -F
#	IGNORE_ERROR: 1
#
This-Is-Invalid-Attribute = "Tapioca Queijo"
