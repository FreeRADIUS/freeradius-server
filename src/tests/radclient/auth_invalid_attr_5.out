(0) Error parsing "src/tests/radclient/auth_invalid_attr_5.txt": Unknown attribute  "This-Is-Invalid-Attribute" for parent "RADIUS"
radclient: Failed parsing input files
_EXIT(1) CALLED src/bin/radclient.c[1429]
